tests/components/device_tracker/test_init.py

```diff
@@ -1,27 +1,30 @@
 """The tests for the device tracker component."""
 # pylint: disable=protected-access
+import asyncio
 import json
 import logging
+import os
 import unittest
-from unittest.mock import call, patch
 from datetime import datetime, timedelta
-import os
+from unittest.mock import call, patch
 
-from homeassistant.core import callback
```

Comment:
Please revert all the changes to the import back so I can see what imports have changed related to this code change.